watcher/standard/src/main/java/io/fabric8/maven/watcher/standard/SpringBootWatcher.java

```diff
@@ -70,8 +80,46 @@ public void watch(List<ImageConfiguration> configs, Set<HasMetadata> resources,
 
         new PodLogService(logContext).tailAppPodsLogs(kubernetes, getContext().getNamespace(), resources, false, null, true, null, false);
 
+        String url = getServiceExposeUrl(kubernetes, resources);
+        if (url == null) {
+            url = getPortForwardUrl(resources);
```

Comment:
[3] Actually we should really think about removing this as it feels wrong to me to have a service only for this purpose. Using a pod directly make much more sense imo. I also have no idea how this works when there are multiple pods btw. But lets discuss this in a separate PR